### ISSUE TITLE
After track playback, wrong stopping point is reported.

### ISSUE THREAD

**Issue by csaavedra** [patrickkfkan/volumio-jellyfin#6]
1. Play an album
2. After the first song ends, check the Jellyfin logs, you'll find something like this:

```
[2023-06-15 11:35:34.938 +03:00] [INF] User "claudio" is now listening to '"Fire"' from album '"Magnificent City Instrumentals"' by '"RJD2"'
[2023-06-15 11:39:44.409 +03:00] [INF] Playback stopped reported by app "jellyfin" "1.0.4" playing "Fire". Stopped at "45" ms
```

What happens is that when the notification is sent to the server that playback for the song ended, the position of the song /that started playing afterwards/ is reported, instead of the position at which the first song ended playback. This causes jellyfin plugins for last.fm and listenbrainz to break, as they use this information to decide whether a track has played long enough to be submitted to either service.

This bug doesn't happen with the last track in the album, as when it stops, there is no new track playing.

Finamp has the same bug, FWIW: https://github.com/jmshrv/finamp/issues/421

**Comment by csaavedra** [comment]
For comparison this is the output when playing from jellyfin's web UI:

```
[2023-06-15 11:52:26.903 +03:00] [INF] "csaavedra" is now playing artist="RJD2", track="All for U", album="Magnificent City Instrumentals"
[2023-06-15 11:52:27.102 +03:00] [INF] User "claudio" is now listening to '"All for U"' from album '"Magnificent City Instrumentals"' by '"RJD2"'
[2023-06-15 11:55:54.622 +03:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.8.10" playing "All for U". Stopped at "208160" ms
[2023-06-15 11:55:54.627 +03:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.8.10" playing "All for U". Stopped at "0" ms
```

There is a bogus 0 ms message, but before that a message with the correct length of the track.

**Comment by patrickkfkan** [comment]
Thanks for the detailed report. Fixed in v1.0.6 - currently in beta channel of Volumio plugin store.

To install from beta channel (all steps mandatory):

1. Uninstall current Jellyfin plugin
2. Restart Volumio
3. Enable "Plugins Test Mode" in `http://<your volumio address>/dev`
4. In Volumio plugin store, click "Details" under Jellyfin plugin, then "Install v1.0.6 (beta)"

**Comment by csaavedra** [comment]
Thanks Patrick for the prompt fix!
